pkg/storage/driver/sql.go

```diff
@@ -681,3 +685,13 @@ func (s *SQL) getReleaseCustomLabels(key string, namespace string) (map[string]s
 
 	return filterSystemLabels(labelsMap), nil
 }
+
+// Rebuild system labels from release object
+func getReleaseSystemLabels(rls *rspb.Release) map[string]string {
```

Comment:
IMHO, this should be part of the `util.go` as all other functions are shared there.

Reply:
The reason why I've placed this code in SQL driver is that as far as I understand it's only useful for SQL driver.
To be honest, it's actually just a workaround to rebuild metadata (which in SQL driver is stored separately).
The reason why I use such an approach to rebuild metadata from the release is that I'm not sure that rewriting SQL queries is worth it, cause all useful fields (all metadata except revision, createdAt, modifiedAt) can be already extracted from the release body, so it seems it doesn't worth it to extract it from DB two times.

@FrenchBen what do you think:
- Is it worth it to rewrite SQL driver code to receive full metadata from corresponding columns instead of rebuilding only the useful part of metadata from the release body?
- Is it worth it to move this helper to util if it shouldn't be used in any other drivers (cause they already read this metadata from storage)?

Reply:
@joejulian @mattfarina just pinging a couple other maintainers here so this can be approved as it's fixing a pretty critical bug

Reply:
I think this is a reasonable reason to leave it put.